stable/youtubedl-material/1.0.0/test_values.yaml

```diff
@@ -0,0 +1,53 @@
+# # Default values for YoutubeDL-Material.
+
+image:
+  repository: tzahi12345/youtubedl-material
+  pullPolicy: IfNotPresent
+  tag: 4.2
+strategy:
+  type: Recreate
+
+# Configure the Security Context for the Pod
+podSecurityContext:
+  runAsNonRoot: false
+  runAsUser: 0
+  runAsGroup: 0
+  fsGroup: 0
+  supplementalGroups: [24, 44, 107]
+
+services:
+  main:
+    enabled: true
+    port:
+      port: 17442
+
+envTpl:
+# Permissions Settings
+  UID: "{{ .Values.PUID }}"
+  GID: "{{ .Values.PGID }}"
+
+env: {}
+  # TZ: UTC
+
+persistence:
+  config:
+    enabled: true
+    mountPath: "/app/appdata"
+    emptyDir: true
+    accessMode: ReadWriteOnce
+    size: 1Gi
+    storageClass: ""
+
+## TrueCharts Values
+
+ingress:
+  main:
+    enabled: true
+    # Used when including ingress using {{ include "common.ingress" . }}
+    type: "HTTP"
+    entrypoint: "websecure"
+    certType: ""
+    annotations: {}
+    hosts:
+      - host: chart-example.local
+        path: /
```

Comment:
don't fully setup ingress in tests.

Reply:
Hmm i copied that from jackett, now looking on other apps, there is no test at all for ingress, should i remove it completely?

Reply:
No. Ingress.main.enabled: false

Reply:
(like the rest afaik.)
THough removal should also be fine :)